packages/compass-e2e-tests/helpers/commands/index.js

```diff
@@ -4,28 +4,21 @@
  * @param {ExtendedApplication} app
  */
 exports.addCommands = function (app) {
-  app.client.addCommand('waitForElement', require('./wait-for-element')(app));
```

Comment:
Removed these because I realised webdriver already has lots of waitForX methods.